Homeworks/homework_11.py

```diff
@@ -0,0 +1,85 @@
+"""
+Homework 11.1.
+
+Ваша команда та ви розробляєте систему входу для веб-додатка,
+і вам потрібно реалізувати тести на функцію
+для логування подій в системі входу.
+Дано функцію, напишіть набір тестів для неї.
+"""
+
+import logging
+
+import pytest
+
+
+def read_file(log_level, message):
+    """
+    Read file.
+
+    :param log_level: logging level
+    :param message: message text
+    """
+    expected_result = f'{log_level} - {message}'
+    with open('homework11_login_system.log', 'r') as file:
+        text_in_file = file.readlines()
+        len_text_file = len(text_in_file)
+        actual_result = text_in_file[len_text_file - 1]
+        actual_result = ' - '.join(actual_result.split(' - ')[1:])
+        assert actual_result[:-1] == expected_result
+
+
+def log_event(username: str, status: str):
```

Comment:
This function should be in separate file ( cause this is developers code)  and you need to do import